articles/machine-learning/v1/azure-machine-learning-release-notes.md

```diff
@@ -19,6 +19,17 @@ In this article, learn about Azure Machine Learning Python SDK releases.  For th
 __RSS feed__: Get notified when this page is updated by copying and pasting the following URL into your feed reader:
 `https://learn.microsoft.com/api/search/rss?search=%22Azure+machine+learning+release+notes%22&locale=en-us`
 
+## 2023-11-13
+  + **azureml-automl-core, azureml-automl-runtime, azureml-contrib-automl-dnn-forecasting, azureml-train-automl-client, azureml-train-automl-runtime, azureml-training-tabular**
+    +  statsmodels, pandas and scipy were upgraded to versions 1.13, 1.3.5 and 1.10.1 - fbprophet 0.7.1 was replaced by prophet 1.1.4 When loading a model in a local environment, the versions of these packages should match what the model was trained on.
+  + **azureml-core, azureml-pipeline-core, azureml-pipeline-steps**
+    + [team name] -> AzureML-Pipeline [Put release notes here] -> Add a warning for the `init_scripts` parameter in the Databricks step, alerting you to its upcoming deprecation.
```

Comment:
@isaudagar, should the `[team name]` and `[Put release notes here]` values be replaced with different text? Maybe these are left over from template text?

Reply:
Connected and resolved comments.